python/lsst/obs/base/gen2to3/repoConverter.py

```diff
@@ -425,30 +425,15 @@ def findDatasets(self):
         self.task.log.info("Adding special datasets in repo %s.", self.root)
         for dataset in self.iterDatasets():
             assert len(dataset.refs) == 1
-            self._fileDatasets[dataset.refs[0].datasetType].append(dataset)
+            self._fileDatasets[dataset.refs[0].datasetType][None].append(dataset)
```

Comment:
Maybe add a short comment here explaining that the None is (I think) the calib date?